py/desispec/qproc/qfiberflat.py

```diff
@@ -7,28 +7,32 @@
 from desispec.qproc.qframe import QFrame
 from desispec.fiberflat import FiberFlat
 
-def qproc_apply_fiberflat(qframe,fiberflat) :
+def qproc_apply_fiberflat(qframe,fiberflat,return_flat=False) :
```

Comment:
As long as you are updating this function, please also add a docstring.  e.g. I was initially confused about what flat would be returned given that a fiberflat is an input.

Reply:
done (not sure how to minimize the number of spam emails to colleagues but of this github feature to comment in the code)